test/integration/main_test.go

```diff
@@ -140,6 +142,11 @@ func HyperVDriver() bool {
 	return strings.Contains(*startArgs, "--driver=hyperv") || strings.Contains(*startArgs, "--vm-driver=hyperv")
 }
 
+// KVM returns true is is KVM driver
+func KVMDriver() bool {
+	return strings.Contains(*startArgs, "--driver=kvm") || strings.Contains(*startArgs, "--vm-driver=kvm") || strings.Contains(*startArgs, "--driver=kvm2") || strings.Contains(*startArgs, "--vm-driver=kvm2")
```

Comment:
I know we use this for other drivers, but this will not work with "--driver kvm". We cannot fix this now, but we should change the way we run the tests, test configuration should come from a configuration file:

    $ cat test.yaml
    configs:
    - name: kvm
      os: linux
      driver: kvm
      auto-update-drivers: false
    - name: vfkit
      os: darwin
      driver: vfkit
      network: vmnet-shared  

    $ make integration CONFIG=kvm

Reply:
good call